docs/crio.conf.5.md

```diff
@@ -167,8 +167,8 @@ The `crio.runtime` table contains settings pertaining to the OCI runtime used an
 **container_attach_socket_dir**="/var/run/crio"
   Path to directory for container attach sockets.
 
-**bind_mount_prefix**=""
-  The prefix to use for the source of the bind mounts.
+**bind_mount_prefix**="prefix"
+  A prefix to use for the source of the bind mounts. This option would be useful if you were running CRI-O in a container and had the / on the host mounted as /host on your container. Then if you ran CRI-O with the --bind-mount-prefix=/host option, CRI-O would add /host to any bind mounts it is handed over CRI. If Kubernetes asked to have /var/lib/foobar bind mounted into the container, then CRI-O would bind mount /host/var/lib/foobar. Since CRI-O itself is running in a container with / or the host mounted on /host, the container would end up with /var/lib/foobar from the host mounted in the container rather then /var/lib/foobar from the CRI-O container.
```

Comment:
I'm not a fan of the word "you" in man pages.  Suggest:

"A prefix to use for the source of the bind mounts. This option would be useful when running CRI-O in a container and the / directory on the host is mounted as /host in the container. Then if CRI-O runs with the --bind-mount-prefix=/host option, CRI-O would add the /host directory to any bind mounts it hands over to CRI. If Kubernetes asked to have /var/lib/foobar bind mounted into the container, then CRI-O would bind mount /host/var/lib/foobar. Since CRI-O itself is running in a container with / or the host mounted on /host, the container would end up with /var/lib/foobar from the host mounted in the container rather than /var/lib/foobar from the CRI-O container."